src/client/components/App/App.tsx

```diff
@@ -1,38 +1,61 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
+import { useSelector, Provider } from 'react-redux';
```

Comment:
bumped all the provider components (for Redux + the custom WebSockets, and the GameManager providers) into App